src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.cs

```diff
@@ -14,6 +14,7 @@ namespace System.Runtime.Intrinsics.Arm
     [Intrinsic]
     [CLSCompliant(false)]
     [System.Runtime.Versioning.RequiresPreviewFeaturesAttribute("Sve is in preview.")]
+    [Experimental(diagnosticId: "Sve")]
```

Comment:
I just realized that perhaps the [RequiresPreviewFeaturesAttribute](https://learn.microsoft.com/en-us/dotnet/api/system.runtime.versioning.requirespreviewfeaturesattribute?view=net-8.0) should be sufficient and we might not need [Experimental](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-12.0/experimental-attribute), which is used to report errors if the caller doesn't use pragma to suppress it.  Sorry for the confusion.

Reply:
No worries - removed.